packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembers.tsx

```diff
@@ -20,19 +21,31 @@ import { mqu } from '../../styledSystem';
 import { ActiveMembersList } from './ActiveMembersList';
 import { MembersActionsRow } from './MembersActions';
 import { MembershipWidget } from './MembershipWidget';
+import { MembersSearch } from './MembersSearch';
 import { OrganizationMembersTabInvitations } from './OrganizationMembersTabInvitations';
 import { OrganizationMembersTabRequests } from './OrganizationMembersTabRequests';
 
+export const ACTIVE_MEMBERS_PAGE_SIZE = 10;
+
 export const OrganizationMembers = withCardStateProvider(() => {
   const { organizationSettings } = useEnvironment();
   const card = useCardState();
   const canManageMemberships = useProtect({ permission: 'org:sys_memberships:manage' });
   const canReadMemberships = useProtect({ permission: 'org:sys_memberships:read' });
   const isDomainsEnabled = organizationSettings?.domains?.enabled && canManageMemberships;
+
+  const [query, setQuery] = useState<string>();
```

Comment:
Can we just use empty string as the default value? 
```suggestion
  const [query, setQuery] = useState('');
```

Reply:
I've initialized it as `undefined` to initially filter out that query since FAPI returns 400 status code for an empty `query` param 

I think we can remove due to the `query || undefined` on line 46